src/Libraries/RevitNodes/GeometryConversion/ProtoToRevitMesh.cs

```diff
@@ -102,35 +101,34 @@ public static class ProtoToRevitMesh
              ElementId MaterialId = null,
             bool performHostUnitConversion = true)
         {
-         
+
             var verts = mesh.VertexPositions;
-            var indicies = mesh.FaceIndices;
+            var indices = mesh.FaceIndices;
 
-            var currentVerts = new List<Autodesk.DesignScript.Geometry.Point>();
             var tsb = new TessellatedShapeBuilder();
             tsb.OpenConnectedFaceSet(false);
 
-            foreach (var f in indicies)
+            for (int faceindex = 0; faceindex < indices.Count(); faceindex++)
```

Comment:
If the length of the array is not changing, you can do this to remove the cost of `indices.Count()`:

```csharp
for (var i = 0, l = indices.Count(); i < l; i++){
...
}
```